pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -590,105 +596,111 @@ message ContainerConfig {
     // container logs are under active discussion in
     // https://issues.k8s.io/24677. There *may* be future change of direction
     // for logging as the discussion carries on.
-    optional string log_path = 11;
+    string log_path = 11;
 
     // Variables for interactive containers, these have very specialized
     // use-cases (e.g. debugging).
     // TODO: Determine if we need to continue supporting these fields that are
     // part of Kubernetes's Container Spec.
-    optional bool stdin = 12;
-    optional bool stdin_once = 13;
-    optional bool tty = 14;
+    bool stdin = 12;
+    bool stdin_once = 13;
+    bool tty = 14;
 
     // Configuration specific to Linux containers.
-    optional LinuxContainerConfig linux = 15;
+    LinuxContainerConfig linux = 15;
 }
 
 message CreateContainerRequest {
     // ID of the PodSandbox in which the container should be created.
-    optional string pod_sandbox_id = 1;
+    string pod_sandbox_id = 1;
     // Config of the container.
-    optional ContainerConfig config = 2;
+    ContainerConfig config = 2;
     // Config of the PodSandbox. This is the same config that was passed
     // to RunPodSandboxRequest to create the PodSandbox. It is passed again
     // here just for easy reference. The PodSandboxConfig is immutable and
     // remains the same throughout the lifetime of the pod.
-    optional PodSandboxConfig sandbox_config = 3;
+    PodSandboxConfig sandbox_config = 3;
 }
 
 message CreateContainerResponse {
     // ID of the created container.
-    optional string container_id = 1;
+    string container_id = 1;
 }
 
 message StartContainerRequest {
     // ID of the container to start.
-    optional string container_id = 1;
+    string container_id = 1;
 }
 
 message StartContainerResponse {}
 
 message StopContainerRequest {
     // ID of the container to stop.
-    optional string container_id = 1;
+    string container_id = 1;
     // Timeout, in seconds, to stop the container.
-    optional int64 timeout = 2;
+    int64 timeout = 2;
 }
 
 message StopContainerResponse {}
 
 message RemoveContainerRequest {
     // ID of the container to remove.
-    optional string container_id = 1;
+    string container_id = 1;
 }
 
 message RemoveContainerResponse {}
 
 enum ContainerState {
-    CONTAINER_CREATED = 0;
-    CONTAINER_RUNNING = 1;
-    CONTAINER_EXITED  = 2;
-    CONTAINER_UNKNOWN = 3;
+    CONTAINER_CREATED      = 0;
```

Comment:
This is because of removing an extra state in the old commits. Will remove those extra spaces.